dkeyserver/App/main.cpp

```diff
@@ -174,7 +241,7 @@ int main()
         return -1;
     }
 
-    ret = sgx_set_up_tls_server(g_enclave_id, &ret, s_port);
+    ret = sgx_set_up_tls_server(g_enclave_id, &ret, s_port, target_ip_addr.c_str(), target_port);
```

Comment:
add some comments, to describe the overall flow about how to get the domainkey.

Reply:
this will be changed on another pr.